yugabyte/yugabyte-db
# Master UI shows incorrect number of tablets for colocated tables #3674

Fix for issue: https://github.com/yugabyte/yugabyte-db/issues/3674

Test plan: 

1. run yb-ctl to create a cluster with RF=1
2. Create a database with colocated=true and three colocated tables
3. Go  to  http://127.0.0.1:7000/tablet-servers
4. One should see...
![image](https://user-images.githubusercontent.com/62734812/84977238-b7236580-b0de-11ea-98f0-e6818c083b99.png)

Please review.

Thanks
